internal/httputil/client.go

```diff
@@ -17,14 +16,9 @@ import (
 // ErrTokenRevoked signifies a token revokation or expiration error
 var ErrTokenRevoked = errors.New("token expired or revoked")
 
-var httpClient = &http.Client{
-	Timeout: time.Second * 5,
-	Transport: &http.Transport{
-		Dial: (&net.Dialer{
-			Timeout: 2 * time.Second,
-		}).Dial,
-		TLSHandshakeTimeout: 2 * time.Second,
-	},
+// DefaultClient avoids leaks by setting an upper limit for timeouts.
+var DefaultClient = &http.Client{
```

Comment:
All we really care about is the [total timeout](https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/).